platform/src/test/kotlin/org/stellar/anchor/Sep1ServiceTest.kt

```diff
@@ -66,9 +69,58 @@ class Sep1ServiceTest {
     mockServer.start()
     val mockAnchorUrl = mockServer.url("").toString()
     mockServer.enqueue(MockResponse().setBody(stellarToml))
-
     val config = PropertySep1Config(true, TomlConfig(URL, mockAnchorUrl))
     sep1 = Sep1Service(config)
     assertEquals(sep1.stellarToml, stellarToml)
   }
+
+  // this test is not expected to raise an exception. given the re-direct to a malicious
+  // endpoint still returns a 200 the exception will be raised/obfuscated
+  // when the toml is parsed.
+  @Test
+  fun `getStellarToml fetches invalid data during malicious re-direct`() {
```

Comment:
Nit: I think the intention is to make it obvious this _can_ happen, but it seems a bit weird given that this is something we would want to prevent. Maybe we could change the test name to something like `getStellarToml fetches data during re-direct`. What do you think?